ocamldoc/odoc_messages.ml

```diff
@@ -200,6 +200,7 @@ let merge_param = ('p', "merge @param")
 let merge_raised_exception = ('e', "merge @raise")
 let merge_return_value = ('r', "merge @return")
 let merge_custom = ('c', "merge custom @-tags")
+let merge_alert = ('x', "merge alerts")
```

Comment:
'x' for `alerts`?

Reply:
Every letters of `alerts` are taken except `t` and I thought alerts are some kind of next-gen tags and needed a fancy letter.

Reply:
Fine with me.